prosody/Dockerfile

```diff
@@ -1,10 +1,30 @@
 FROM jitsi/base
 
+ADD https://prosody.im/files/prosody-debian-packages.key /tmp/key
+
+ADD https://raw.githubusercontent.com/jitsi/jitsi-meet/master/resources/prosody-plugins/token/util.lib.lua /prosody-plugins/token/util.lib.lua
+ADD https://raw.githubusercontent.com/jitsi/jitsi-meet/master/resources/prosody-plugins/mod_token_verification.lua /prosody-plugins/mod_token_verification.lua
+ADD https://raw.githubusercontent.com/jitsi/jitsi-meet/master/resources/prosody-plugins/mod_auth_token.lua /prosody-plugins/mod_auth_token.lua
+
 RUN \
 	apt-dpkg-wrap apt-get update && \
-	apt-dpkg-wrap apt-get install -t stretch-backports -y prosody && \
+	apt-dpkg-wrap apt-get install -y gnupg lua5.2 liblua5.2-dev lua-expat lua-socket lua-filesystem ssl-cert luarocks gcc libssl1.1 libssl1.0-dev git
+
+RUN \
+	echo deb http://packages.prosody.im/debian stretch main | tee -a /etc/apt/sources.list && \
+	apt-key add /tmp/key && \
+	apt-dpkg-wrap apt-get update && \
+	apt-dpkg-wrap apt-get install -y prosody && \
 	apt-cleanup && \
-	rm -rf /etc/prosody
+	rm -rf /etc/prosody /tmp/key
+
+RUN \
+	luarocks install luaossl && \
+	luarocks install luajwt && \
+	luarocks install luajwtjitsi && \
+	luarocks install jwt-jitsi && \
+	luarocks install luasec && \
```

Comment:
Let's use the debian package: https://packages.debian.org/source/stretch/lua-sec